FolderDelete.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec 12 15:43:17 2019
+Created on Thu Dec 12 13:55:04 2019
+This code will read the folder directory from an Excel spreadsheet and delete 
+the folder, if empty.
+@author: ad414d
+"""
+
+import os
+from openpyxl import load_workbook
+from openpyxl.styles import PatternFill
+
+#Define variable for the TreeSize workbook.
+wb=load_workbook('C:\\aaTanker\TreeSize\jrl me folder - new.xlsx')
+
+#Define variable for the worksheet.
+sheet = wb['Custom Search']
+
+for i in range(5, sheet.max_row):
+    #Check if the file exists, then delete it.
+    if os.path.exists(sheet.cell(row=i,column=3).value):
+        #print("Path exists.")
+        #print(sheet.cell(row=685,column=3).value+sheet.cell(row=685,column=2).value)
+        try:
+            os.rmdir(sheet.cell(row=i,column=3).value)
+        except OSError:
+            is_empty = False

```

Comment:
Do you ever set is_empty to True? If not, it will always be false (or at least, it will stay false starting the first time you set it to false).

Reply:
I do not, and tried to delete lines 27 and 28.  The code showed an error, so I left those lines in.  I'm not sure the best way to handle checking if a folder is empty and moving on if it is not which is what these lines are doing.

Reply:
Oh, you want to ignore exceptions in rmdir. The correct incantation is
```
except OSError:
  pass
```
(`pass` means "do nothing").